Changelog.md

```diff
@@ -1,8 +1,52 @@
 # Changelog
 
-## Omigami 0.3.0
-### Summary of changes
-- to be summarized
-  
-### Details
-- to be detailed
\ No newline at end of file
+## Releases
+***
+v0.3.0
+***
```

Comment:
I think these 4 lines should be ## v0.3.0. The rendering looks weird

Reply:
you mean we need 4 starts to draw a line properly?

Reply:
ok I updated headings